account_bank_statement_import_online/models/online_bank_statement_provider.py

```diff
@@ -225,7 +238,9 @@ def _pull(self, date_since, date_until):
                     )
                 filtered_lines = []
                 for line_values in lines_data:
-                    date = fields.Datetime.from_string(line_values['date'])
+                    date = line_values['date']
+                    if isinstance(date, str):
```

Comment:
I think don't need here.

but add a check to check that line_values['date'] is a datetime with no tz (like datime used in odoo).

Reply:
It's done to adapt to some providers that have only date and can send that as string

Reply:
you don't need to adapt `from_string` adapt automaitcely `datetime` or `str` and check that there are no timezone.
https://github.com/odoo/odoo/blob/12.0/odoo/fields.py#L1741
all date should be pass by `from_string`

Reply:
That's not for adapting for Odoo, but parsing for processing within the module to allow provider to provide either date or datetime or string.